ulib/LowStar.Monotonic.Buffer.fsti

```diff
@@ -1185,7 +1255,7 @@ val modifies_fresh_frame_popped
     modifies s h0 h3 /\
     (HS.get_tip h3) == HS.get_tip h0
   ))
-  [SMTPat (HS.fresh_frame h0 h1); SMTPat (HS.popped h2 h3); SMTPat (modifies s h0 h3)]
+  //[SMTPat (HS.fresh_frame h0 h1); SMTPat (HS.popped h2 h3); SMTPat (modifies s h0 h3)]
```

Comment:
so now the recommended style is to call this lemma directly when you have a non-trivial modifies clause inside the push/pop? can this be documented at least with a comment?

Reply:
Sure, documenting it is a good idea. Though with these changes, we are not calling this lemma anywhere now.

Should I just add a comment in the buffer .fsti or CHANGES.md?

Reply:
so no client should ever have to call this lemma manually?

fsti is better, I think -- more discoverable

Reply:
Yes, at least that's the hope. If we find an example where we are having to call it explicitly, we should try to fix the library. I will add a comment. Thanks.